package-support-policy.md

```diff
@@ -0,0 +1,67 @@
+
+These are packages that originally had their source code hosted in branches that are already out of support, but the packages themselves are still in-support, because they target frameworks that are still supported. Some examples:
+
+- [System.Xml.XPath.XmlDocument](https://www.nuget.org/packages/System.Xml.XPath.XmlDocument) - Its source code used to be hosted in the .NET Core 1.1 branch.
+- [System.Buffers](https://www.nuget.org/packages/System.Buffers) - Its source code used to be hosted in the .NET Core 2.1 branch.
+- [Microsoft.Bcl.HashCode](https://www.nuget.org/packages/Microsoft.Bcl.HashCode) - Its source code used to be hosted in the.NET Core 3.1 branch.
+- [System.Net.WebSockets.WebSocketProtocol](https://www.nuget.org/packages/System.Net.WebSockets.WebSocketProtocol) - Its source code used to be hosted in the .NET 5.0 branch.
+
+## End of support
+
+A package will be supported for as long as it targets frameworks that are still supported. For example: [System.Numerics.Vectors](https://www.nuget.org/packages/System.Json#supportedframeworks-body-tab) will is supported because it provides an implementation on .NET Framework 4.6.2, which is still in support.
+
+## Reporting issues
+
+If you find an issue in an assembly that is hosted in this repo, please report it in our central repo: https://github.com/dotnet/runtime/issues/new/choose.
+
+## Should I consume this package? Which version should I consume?
+
+Please refer to the table below to:
+
+- Learn if a package is still supported and in which target frameworks.
+- Learn what action you should take if you're actively consuming one of these packages.
+
+| Package                                 | Still supported? | Reasons                                         | Actions                                |
+|-----------------------------------------|------------------|-------------------------------------------------|----------------------------------------|
+| Microsoft.Bcl.HashCode                  | Yes              | Provides the implementation of [`HashCode`](https://learn.microsoft.com/en-us/dotnet/api/system.hashcode) for .NET Framework. | <ul><li>.NET Framework 4.6.2+: Update to latest version of the package.</li><li>.NET 6+: No need to reference the package. The type is part of the shared framework.</li></ul> |
+| Microsoft.CSharp                        | Limited          | Contains an implementation but it is unused in all the currently supported frameworks. | Only reference the package if you're targeting .NET Standard 2.0 and you need to use the Microsoft.CSharp. The implementation found in the package won't be used, but will make sure the namespace is found. The in-box implementation will be used instead. This package won't take any fixes. |
```

Comment:
Can you somehow make more of the rows use consistent verbiage or style to make it clear how things differ?  Try to limit the actions column to just the framework and the recommendation (remove, update, or avoid use).

```suggestion
| Microsoft.CSharp                        | Limited          | Only used as a reference on .NETStandard 2.0. Other uses are not supported. | <ul><li>.NETStandard 2.0: reference the latest package to use types in the `Microsoft.CSharp` namespace </li><li> All other supported frameworks: remove, the library is part of the framework. |
```

Reply:
Fixed